CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## UNRELEASED
 * Feature - Support for provisioning Tasks with ENIs
+* Enhancement - Retry failed container image pull operations.
```

Comment:
This also added another feature that add customized route in the container namespace for task using `awsvpc`.  If I read the code correctly, this only can be configured by the config file, would it be more convenient to use environment variable. Also we may need to check the route before adding it, as the task traffic inside the container namespace can be changed by adding some route?

Reply:
You're right; I had intended to make it configurable by environment variable as well but apparently never wrote the code.  I'll do that.

I don't think we need to check the route.  This is a sufficiently advanced, optional feature that is used in fairly narrow use-cases.